src/components/AutoSuggest/index.js

```diff
@@ -241,6 +242,7 @@ const AutoSuggest = ({
           ref={inputRef}
           disabled={disabled}
           id={id}
```

Comment:
How about preventing auto-complete here, since it interferes with our own version of autocomplete/suggest? 

![Screenshot 2021-03-22 at 14 37 02](https://user-images.githubusercontent.com/10672773/111998360-38030800-8b1c-11eb-9496-01f939f2e95d.png)


```suggestion
          id={id}
          autoComplete="off"
```